docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -133,7 +133,17 @@ Here are a few examples of permission combinations that satisfy Rancher's needs:
 
 :::
 
-#### 4. Copy Azure Application Data
+#### 4. Allow Public Client Flows
+
+To login from Rancher CLI you will need to allow Public Client Flows.
```

Comment:
Per the check, the word "will" should be removed.

Reply:
Thanks @crobby, before fixing it, is the rest ok? :D